qmail-rspawn.c

```diff
@@ -84,7 +95,9 @@ char *s; char *r; int at;
  int f;
  char *(args[5]);
 
- args[0] = "qmail-remote";
+ setup_qrargs();
+
+ args[0] = qrargs[0];
```

Comment:
setup_qrargs() could just return the value. Then qrargs could be a static variable just in that function.

Reply:
Agreed. When I wrote the patch, I was trying to make it look as much as possible like [the QMAILQUEUE patch](http://qmailorg.schmonz.com/qmailqueue-patch). I'm not sure why I thought that was important at the time.

Reply:
@schmonz I can see that you have done the change @DerDakon requested. I believe you can hit the 'Resolve conversation' to indicate that you have made the change.

Reply:
You're right, except that I haven't lifted a finger to do anything new with this code yet :-)  Too far in the future, and too curious to see your alternative.